docs/configuration.md

```diff
@@ -42,27 +42,28 @@ Unless you need to set a non-default value, it is recommended to only populate o
-| `ETH_CALL_CACHE_TTL`              | "200"         | Maximum time in ms to cache an eth_call response.   |
-| `ETH_CALL_DEFAULT_TO_CONSENSUS_NODE `              | "false"       | Flag to set if eth_call logic should first query the mirror node.   |
-| `ETH_GET_LOGS_BLOCK_RANGE_LIMIT`  | "1000"        | The maximum block number range to consider during an eth_getLogs call.   |
-| `FEE_HISTORY_MAX_RESULTS`         | "10"          | The maximum number of results to returns as part of `eth_feeHistory`.   |
-| `GAS_PRICE_TINY_BAR_BUFFER`       | "10000000000" | The additional buffer range to allow during a relay precheck of gas price. This supports slight fluctuations in network gasprice calculations.   |
-| `LIMIT_DURATION`                  | "60000"       | The maximum duration in ms applied to IP-method based rate limits.   |
-| `MIRROR_NODE_LIMIT_PARAM`         | "100"         | The mirror node custom limit value to be set on GET requests. This optimizes the flow to reduce the number of calls made to the mirror node by setting a limit larger than it's default limit.   |
-| `MIRROR_NODE_RETRIES`             | "3"           | The maximum number of retries on a GET request to the mirror node when an acceptable error code is returned.   |
-| `MIRROR_NODE_RETRY_DELAY`         | "250"         | The delay in ms between retry requests.   |
-| `MIRROR_NODE_RETRIES_DEVMODE`     | "5"           | The maximum number of retries on a GET request to the mirror node when an acceptable error code is returned in dev mode.   |
-| `MIRROR_NODE_RETRY_DELAY_DEVMODE` | "200"         | The delay in ms between retry requests in dev mode.   |
-| `MIRROR_NODE_URL`                 | ""            | The Mirror Node API endpoint. Official endpoints are Previewnet (https://previewnet.mirrornode.hedera.com), Testnet (https://testnet.mirrornode.hedera.com), Mainnet (https://mainnet-public.mirrornode.hedera.com). See [Mirror Node REST API](https://docs.hedera.com/hedera/sdks-and-apis/rest-api)   |
-| `SDK_REQUEST_TIMEOUT`             | "10000"       | The complete timeout for running the SDK `execute()` method. This controls the GRPC channel timeout config when querying with network nodes.   |
-| `TIER_1_RATE_LIMIT`               | "100"         | Maximum restrictive request count limit used for expensive endpoints rate limiting.   |
-| `TIER_2_RATE_LIMIT`               | "800"         | Maximum moderate request count limit used for non expensive endpoints.   |
-| `TIER_3_RATE_LIMIT`               | "1600"        | Maximum relaxed request count limit used for static return endpoints.   |
+| Name                                  | Default       | Description                                                                                                                                                                                                                                                                                            |
```

Comment:
This is annoying.  This happened to me too.  Totally legit markdown table formatting made "nicer" - except not really! - by something - spotless perhaps?  Totally gratuitous! Totally unnecessary!

Reply:
yeah, I was in conflict if was better to let the IDE do its thing and format the whole table, I think is nice the end result... but is completely useless for code review 😓 

If we get a +1 on this I might update PR to update only 1 line (addition) and leave the rest of the table as it was...